tools/brokerd/service/win32/CMakeLists.txt

```diff
@@ -29,6 +29,9 @@ add_pstore_tool (pstore-broker-service
     service_base.hpp
     service_installer.cpp
     service_installer.hpp
+    switches.cpp
+    switches.hpp
+    "${pstore_http_fs_source}"
```

Comment:
Cmake somehow tries to find this file before it runs the pstore-genromfs command as defined in the parent cmake file.

Reply:
I'll look into this, thanks.